src/Storage/Storage.Management/Az.Storage.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '1.8.2'
+ModuleVersion = '1.9.1'
```

Comment:
The Reqyired Module should include the latest version of Az.Accounts (1.6.4) I think, otherwise, this looks good